iStore/Home/HomeView.swift

```diff
@@ -17,8 +17,15 @@ struct HomeView: View {
         NavigationView {
             VStack() {
                 LocationButton()
-                
-                ProductsGrid(products: homeViewModel.products)
+                if homeViewModel.isloading {
```

Comment:
Why do you use `homeViewModel` as state not environment  object?

Reply:
- To state the HomeView ownership of the homeViewModel.
- I'm using the homeView only in the HomeView.
- I only read the mode data.
- I would use the Environment object if I use the object in multiple views.